### PR TITLE
ActiveUtils 3.x

### PR DESCRIPTION
**Pull request by wvanbergen**
Prepare for major release.

@ntalbott @jnormore @karlhungus @kmcphillips

### CODE REVIEW

**Comment by wvanbergen** [comment]
I updated the dependency on activesupport to >= 3.2, and updated Travis to tests against a bunch of different active support versions .

File: test/test_helper.rb [hunk 1]
```diff
@@ -5,4 +5,18 @@
...
+
+def suppress_warnings
+  original_verbosity, $VERBOSE = $VERBOSE, nil
+  yield
+ensure
+  $VERBOSE = original_verbosity
+end
+
+class Minitest::Test
+  def assert_deprecation_warning(message=nil)
```
Comment (kmcphillips) [inline test/test_helper.rb]: :+1:

File: .travis.yml [hunk 2]
```diff
@@ -3,7 +3,9 @@ language: ruby
 sudo: false
 
 rvm:
-  - 1.9.3
-  - 2.1.2
+- 1.9
+- 2.0
+- 2.1
+- 2.2
```
Comment (kmcphillips) [inline .travis.yml]: This collection needs to be indented.

**Comment by kmcphillips** [comment]
:+1:

File: .travis.yml (hunk 2 above)
Comment (wvanbergen) [inline .travis.yml]: It's not needed: https://travis-ci.org/Shopify/active_utils/builds/45954898

**Pull request merged**